Orange/canvas/application/errorreporting.py

```diff
@@ -38,6 +38,20 @@ class OWWidget: pass
 log = logging.getLogger()
 
 
+def get_developer_id():
+    """Check if developer provided his name for Sentry reports."""
+    try:
+        from Orange.misc.environ import data_dir_base
+        fname = os.path.join(data_dir_base(), 'Orange', 'developer_id.txt')
```

Comment:
How do you (we) feel about environment variables, like `ORANGE_MACHINE_ID='whatever'`?

https://github.com/biolab/orange3/pull/1929/commits/6e244005c4202b73a8e1248212159aaacf2eff55 much too late proposes `ORANGE_DEBUG` and https://github.com/biolab/orange3/pull/1771/commits/bdf6abc68b0dfd7067d70d77cbd7216283cfae45 introduced `ORANGE_DEPRECATIONS_ERROR`.